go/libkb/login_state.go

```diff
@@ -200,7 +211,10 @@ func (s *LoginState) GetPassphraseStream(ui SecretUI) (pps *PassphraseStream, er
 	if pps != nil {
 		return pps, nil
 	}
-	if err = s.verifyPassphraseWithServer(ui); err != nil {
```

Comment:
Is the code path totally different though?  `loginWithPromptHelper` does a bunch more stuff than `verifyPassphraseWithServer`, doesn't it?